Lib/doctest.py

```diff
@@ -1566,10 +1572,12 @@ def summarize(self, verbose=None):
         summary is.  If the verbosity is not specified, then the
         DocTestRunner's verbosity is used.
         """
+        from _colorize import ANSIColors, can_colorize
+
         if verbose is None:
             verbose = self._verbose
 
-        notests, passed, failed = [], [], []
+        no_tests, passed, failed = [], [], []
```

Comment:
nit (sorry): I'd leave this variable name as it is for now, so that it's really clear from the diff for future readers exactly what needed to change to add colour support here